sov-modules/sov-modules-impl/bank/tests/burn_test.rs

```diff
@@ -81,10 +81,11 @@ fn burn_deployed_tokens() {
 
     let current_total_supply = query_total_supply(&mut working_set);
     // Total supply does not change
```

Comment:
nit: I think this comment is outdated now

Reply:
Fixed